infra/resources/modules/commons/eventhub.tf

```diff
@@ -17,6 +17,36 @@ module "event_hub" {
     },
   ]
 
+  alerts_enabled = true
+  metric_alerts  = {
+    num_of_messages = {
+      aggregation = "Total"
+      metric_name = "IncomingMessages"
+      description = "High volume of messages to ingest, potentially causing delays in sending subscription request events (~1h)."
+      operator    = "LessThanOrEqual"
+      threshold   = 100000
```

Comment:
**note**: This value can be adjusted. It is currently set at this level based on the present consumption rate.